### PR TITLE
Admin notices for checking antagonists

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Because a player came to me recently with concerns someone was doing it midround on a regular basis (they were), and it shouldn't take log diving to figure out if someone is potentially cheating.

Honestly, who is gonna sit around going through logs on a regular basis to check for that?

### CODE REVIEW

**Comment by Shadowlight213** [comment]
:-1: You dont need to log dive. there's an adminlog button already there.
This is already logged. Message is just pointless and you being lazy. Admins are supposed to trust each other, not have every single thing they do be broadcasted.

File: code/game/verbs/who.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 			var/mob/dead/observer/G = src.mob
 			if(!G.started_as_observer)//If you aghost to do this, KorPhaeron will deadmin you in your sleep.
 				log_admin("[key_name(usr)] checked advanced who in-round")
+				message_admins("[key_name_admin(usr)] checked advanced who in-round)")
```
Comment (JJRcop) [inline code/game/verbs/who.dm]: )

**Comment by hornygranny** [comment]
SOMETHING TO HIDE, CITIZEN?

**Comment by MrStonedOne** [comment]
No, no, no. We already have too much lag

**Comment by KorPhaeron** [comment]
@MrStonedOne 

How does this cause lag?

**Comment by MrStonedOne** [comment]
I was watching football and typing that, meant to say spam.

Anywho, admin messaging needs to be a lot more context aware.

Make this only happen if they aren't a ghost or they didn't start as a ghost.

**Comment by optimumtact** [comment]
The proper way to deal with this is to move all admin_log enteries to the DB, then write a page that can show you an overview of an admins audit history, as well as generate alerts to the current headmins when an admin does certain things regularly

game/attack/other logs can remain as standard logs for now.

I'm working on a log parsing tool that could be extended to include admin logs assuming I could get my hands on some example log formats

https://github.com/optimumtact/logdiver

**Comment by Shadowlight213** [comment]
@optimumtact You should be able to generate some logs on a test server yourself.
If you need, I can give you some older logs minus login/out info when I have time.

**Comment by MrStonedOne** [comment]
First things first.

Kor: fix this so it only messages admins if they do it while playing

**Comment by optimumtact** [comment]
TBH we should move all the logs to the db, and make a uniform db structure

**Comment by Razharas** [comment]
Host and admins say its bad - redo
In the other pr too

**Pull request closed without merging**